From a7551d616f86443f4fed8040f288bf33f4085f33 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Wed, 3 Sep 2014 15:15:57 -0400
Subject: [ethernet] bna: remove unnecessary break after return

Message-id: <1409757361-26781-10-git-send-email-ivecera@redhat.com>
Patchwork-id: 91171
O-Subject: [RHEL7.1 PATCH 09/13] bna: remove unnecessary break after return
Bugzilla: 1136912
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

BZ: #1136912

Upstream commit(s):
commit 6644db0c6db0a14b81bc4bba444fdace6ccc2647
Author: Fabian Frederick <fabf@skynet.be>
Date:   Sun Jul 20 16:27:48 2014 +0200

    bna: remove unnecessary break after return

    Signed-off-by: Fabian Frederick <fabf@skynet.be>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/brocade/bna/cna_fwimg.c b/drivers/net/ethernet/brocade/bna/cna_fwimg.c
index 6a68e8d..6f72771 100644
--- a/drivers/net/ethernet/brocade/bna/cna_fwimg.c
+++ b/drivers/net/ethernet/brocade/bna/cna_fwimg.c
@@ -68,10 +68,8 @@ bfa_cb_image_get_chunk(enum bfi_asic_gen asic_gen, u32 off)
  switch (asic_gen) {
  case BFI_ASIC_GEN_CT:
   return (bfi_image_ct_cna + off);
-  break;
  case BFI_ASIC_GEN_CT2:
   return (bfi_image_ct2_cna + off);
-  break;
  default:
   return NULL;
  }
@@ -83,10 +81,8 @@ bfa_cb_image_get_size(enum bfi_asic_gen asic_gen)
  switch (asic_gen) {
  case BFI_ASIC_GEN_CT:
   return bfi_image_ct_cna_size;
-  break;
  case BFI_ASIC_GEN_CT2:
   return bfi_image_ct2_cna_size;
-  break;
  default:
   return 0;
  }
-- 
1.7.1